charts/datahub-executor-worker/values.yaml

```diff
@@ -0,0 +1,104 @@
+global:
```

Comment:
Do we need to put a default image pointing to our ECR here?

Reply:
done